data/remnant/remnant events.txt

```diff
@@ -86,6 +89,17 @@ event "remnant salvage available"
 
 
 
+event "remnant: scanning tolerance"
+	government "Remnant"
+		"penalty for"
+			scan 0
+		"provoked on scan" 0
```

Comment:
> It can't be zeroed out? That's problematic.

What funny is what you're doing here in fact is putting it to true, it just looks at `provoked on scan` and doesn't even consider the 0.

> Is there much code required?

2 lines

Reply:
The possibility to remove provoked on scan could also be added straight to master but since you're the only one using it and idk how much time it would take for it to be merged quick fix as it may be, I just opened it to your branch.
The remnants are the only ones using it anyway.